best-practices.md

```diff
@@ -30,19 +30,13 @@ All rules below should be followed by contributors to this repo. Contributors sh
 - output file names should *always* be determined with either the `outfile_name` parameter or the `prefix` parameter.
   - `outfile_name` should be preferred if no downstream tasks/tools rely on the file name/extension
   - tasks with multiple outputs should always use the `prefix` convention
-- After the input sorting rules in `style-guide.md` have been applied, follow the below rules for further sorting.
+- After the input sorting rules in `sprocket lint` have been applied, follow the below rules for further sorting.
   - "sample" files come before "reference" files
   - If present, `use_all_cores` should be the last `Boolean` in its block
   - the `ncpu` parameter comes before inputs that allocate memory, which come before inputs that allocate disk space
     - This block of 2-3 inputs should come after all other inputs.
 - Most tasks should have a default `maxRetries` of 1
-  - Certain tasks are prone to intermittent failure (often if an internet connection is involved) and can have a higher default `maxRetries`. This value should not exceed 3.
-- There are lower bounds for resource allocation
-  - Memory should be allocated a minimum of 4gb
-  - Disk size should be allocated a minimum of 10gb
-  - If the task is multi-cored, it should use at least 2 cpu by default
```

Comment:
This line (43) should maybe be restored. I think it's generally advisable that if multi-threading is available it should be enabled by default.
I suppose a counterpoint is that most of the samtools tasks are able to run on multiple cores, but all that usually does is multi-thread [de]compression which doesn't see much speed up.

Reply:
I went with a very generic statement, though unfortunately it now shows below this comment.